File: C/1013.c
#include <stdio.h>

int main()
{
    int a, b, c, max;
    scanf("%d %d %d", &a, &b, &c);
    if (b > max)
        max = b;
    if (c > max)
        max = c;
    printf("%d eh o maior\n", max);
    return 0;
}

File: PHP/1013.php
<?php
$x = rtrim(fgets(STDIN));
$a = explode(' ', $x)[0];
$b = explode(' ', $x)[1];
$c = explode(' ', $x)[2];
$max = $a;
if ($b > $max) {
    $max = $b;
}

if ($c > $max) {
    $max = $c;
}

echo $max . " eh o maior" . PHP_EOL;
